src/domain/enquiryService.ts

```diff
@@ -0,0 +1,31 @@
+import VehicleEvent from '../interfaces/VehicleEvent';
+import { validateVehicleEvent, validateResultsEvent } from '../utils/validationService';
+import vehicleQueryFunctionFactory from '../app/vehicleQueryFunctionFactory';
+import testQueryFunctionFactory from '../app/testQueryFunctionFactory';
+import DatabaseService from '../interfaces/DatabaseService';
+import ResultsEvent from '../interfaces/ResultsEvent';
+import VehicleDetails from '../interfaces/queryResults/technical/vehicleDetails';
+import TestRecord from '../interfaces/queryResults/test/testRecord';
+
+const getVehicleDetails = async (
+  event: VehicleEvent,
```

Comment:
I don't think I follow this one, the event is required in the function factory, and even then I couldn't have a function that just takes the DB service because I'd have to `bind` the event to the function to make it available, which is a bit weird. I think?